guide/admin/homeview.md

```diff
@@ -0,0 +1,33 @@
+## User Handbook
+### Changing the Home Page View
```

Comment:
Maybe something like *Switching Home Page display options*? does that sound better you think? I would like to try and keep it at Home Page because that is what it is listed as in the ACP, for sake of consistency.